src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -31,14 +30,17 @@ const CompleteIncident = props => {
 
   // setting state to toggle "editing mode"
   const [editing, setEditing] = useState(false);
+
+  const [deleting, setDeleting] = useState(false);
+
   const [formValues, setFormValues] = useState({});
 
   const oktaAxios = useOktaAxios();
 
   useEffect(() => {
     setFormValues({
       ...incident,
-      tags: incident.tags ? incident.tags.join(", ") : [],
+      tags: incident.tags ? incident.tags.join(', ') : [],
       incident_date: formattedDate,
```

Comment:
Was this necessary?

Reply:
No necessarily, I was just trying to clean up the code a little and make it all look the same